web_external/views/adapters/Adapters.js

```diff
@@ -600,21 +601,36 @@ rendering.geo.WmsRepresentation = rendering.geo.defineMapLayer('wms', function (
 }, rendering.geo.MapRepresentation);
 
 rendering.geo.KtileRepresentation = rendering.geo.defineMapLayer('ktile', function () {
-    this.init = function (container, dataset) {
+    this.init = function (container, dataset, visProperties) {
         var layer = container.createLayer('osm', {
             attribution: null,
             keepLower: false
         });
         var fileId = dataset.get('meta').minerva.original_files[0]._id;
-        var visProperties = dataset.getMinervaMetadata().visProperties;
         var url = getApiRoot() + '/ktile/' + fileId;
-        if (visProperties) {
-            layer.url((x, y, z) => `${url}/${z}/${x}/${y}?palette=${visProperties.colorbrewer}&band=${visProperties.band}&minimum=${visProperties.min}&maximum=${visProperties.max}`);
-        } else {
+        if (_.isEmpty(visProperties)) {
             layer.url((x, y, z) => `${url}/${z}/${x}/${y}`);
+        } else {
+            layer.url((x, y, z) => `${url}/${z}/${x}/${y}?
+            palette=${visProperties.palettable}&band=${visProperties.band}&
+            minimum=${visProperties.min}&maximum=${visProperties.max}`);
```

Comment:
Unbreak this line or change how it is generated.  The extra spaces and newlines mess up the query parameters.  This should work instead:
```
            layer.url((x, y, z) => `${url}/${z}/${x}/${y}?` +
            `palette=${visProperties.palettable}&band=${visProperties.band}&` +
            `minimum=${visProperties.min}&maximum=${visProperties.max}`);
```

Reply:
Oh, yes. I was carried away with the multiline support of template literals.